templates/primary-secondary/primary-secondary.js

```diff
@@ -65,7 +65,7 @@ class TemplatePrimarySecondary extends LitElement {
 			}
 			main {
 				grid-area: primary;
-				overflow-y: auto;
+				overflow-x: hidden;
```

Comment:
This change doesn't seem to be breaking anything in the other browsers (for LTR and RTL)

Reply:
To be clear, is the scrolling "endless", or is it just really far... to the point where our offscreen text would render?

I see why we would add the `overflow-x`, but I'm not sure why we would remove the `overflow-y`.  What's the reasoning for that?

@wongvincent - looks like this `overflow-y` was here (class was changed to name somewhere in the history) since you added this component.  Do you recall why this was included, and do you have concerns about this change?

Reply:
I do not recall. In the initial commit, which was really just testing stuff out, the `overflow-y` CSS property was added along with a `height` CSS property. But then the `height` property was removed and the `overflow-y` remained. The `overflow-y` property may not be required for anything now.

Reply:
@dbatiste yup you're right it is not really an endless scroll - but a really long horizontal scroll. With this change, that horizontal scroll in legacy edge for RTL languages is removed. 

I tried keeping both `overflow-y: auto` as well as `overflow-x: hidden` and it seemed to behave exactly the same as having only then `overflow-x: hidden ` rule